### PR TITLE
New README

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,22 +20,23 @@
   <a href="https://slack.meilisearch.com"><img src="https://img.shields.io/badge/slack-MeiliSearch-blue.svg?logo=slack" alt="Slack"></a>
 </p>
 
-<p align="center">⚡ Lightning Fast, Ultra Relevant, and Typo-Tolerant Search Engine MeiliSearch client written in Go</p>
+<p align="center">⚡ The MeiliSearch API client written for Golang</p>
```
Comment (curquiza) [inline README.md:23]: I also changed this sentence that I found unreadable. WDYT?

Comment (eskombro) [reply README.md:23]: I agree, and just below we explain what MeiliSearch is, just in case!

File: README.md [hunk 2]
```diff
@@ -298,6 +188,19 @@ resp, err := client.Search(indexUID).Search(meilisearch.SearchRequest{
...
+This package only guarantees the compatibility with the [version v0.15.0 of MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.15.0).
+
+## 💡 Learn More
+
+The following sections may interest you:
+
+- **Manipulate documents**: see the [API references](https://docs.meilisearch.com/references/documents.html) or read more about the [document concept](https://docs.meilisearch.com/guides/main_concepts/documents.html).
+- **Search**: see the [API references](https://docs.meilisearch.com/references/search.html) or follow our [guide on search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
+- **Manage the indexes**: see the [API references](https://docs.meilisearch.com/references/indexes.html) or read more about the [index concept](https://docs.meilisearch.com/guides/main_concepts/indexes.html).
+- **Configure the index settings**: see the [API references](https://docs.meilisearch.com/references/settings.html) or follow our [guide on settings parameters](https://docs.meilisearch.com/guides/advanced_guides/settings.html#synonyms).
```
Comment (eskombro) [inline README.md:202]:
```suggestion
- **Manipulate documents**: see the [API references](https://docs.meilisearch.com/references/documents.html) or read more about [documents](https://docs.meilisearch.com/guides/main_concepts/documents.html).
- **Search**: see the [API references](https://docs.meilisearch.com/references/search.html) or follow our [guide on search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
- **Manage the indexes**: see the [API references](https://docs.meilisearch.com/references/indexes.html) or read more about [indexes](https://docs.meilisearch.com/guides/main_concepts/indexes.html).
- **Configure the index settings**: see the [API references](https://docs.meilisearch.com/references/settings.html) or follow our [guide on settings parameters](https://docs.meilisearch.com/guides/advanced_guides/settings.html#synonyms).
```

**Review by eskombro — APPROVED** [review]
I like it. Did a simple suggestion, but otherwise I approve! 👍

File: README.md (hunk 1 above)
Comment (eskombro) [reply README.md:23]: I agree, and just below we explain what MeiliSearch is, just in case!

File: README.md (hunk 2 above)
Comment (curquiza) [reply README.md:202]: Waiting for @bidoubiwa's opinion! 🙂

Comment (eskombro) [reply README.md:202]: Just as a note, notice that I removed the `#synonyms` from the last link, I think it should be removed in any case

Comment (curquiza) [reply README.md:202]: Oh yes! Well spotted!!

Comment (bidoubiwa) [reply README.md:202]: Both are nice but I prefer eskombro's for readability.

File: README.md [hunk 3]
```diff
@@ -103,9 +105,9 @@ func main() {
 }
 ```
 
-With the `updateId`, you can check the status (`processed` or `failed`) of your documents addition thanks to this [method](#update-status).
+With the `updateId`, you can check the status (`enqueued`, `processed` or `failed`) of your documents addition thanks to this [method](https://docs.meilisearch.com/references/updates.html#get-an-update-status).
```
Comment (bidoubiwa) [inline README.md:108]:
```suggestion
With the `updateId`, you can check the status (`enqueued`, `processed` or `failed`) of your documents addition using the [update endpoint](https://docs.meilisearch.com/references/updates.html#get-an-update-status).
```

File: README.md [hunk 4]
```diff
@@ -20,22 +20,23 @@
   <a href="https://slack.meilisearch.com"><img src="https://img.shields.io/badge/slack-MeiliSearch-blue.svg?logo=slack" alt="Slack"></a>
 </p>
 
-<p align="center">⚡ Lightning Fast, Ultra Relevant, and Typo-Tolerant Search Engine MeiliSearch client written in Go</p>
+<p align="center">⚡ The MeiliSearch API client written for Golang</p>
 
-**MeiliSearch Go** is a client for **MeiliSearch** written in Go. **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, and synonyms are provided out-of-the-box.
+**MeiliSearch Go** is a client for **MeiliSearch** written in Go. **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, facets and synonyms are provided out-of-the-box.
```
Comment (bidoubiwa) [inline README.md:25]: I read that MeiliSearch was written in Go 🤔. Not sure if it is me or if the english is weird.

Comment (curquiza) [inline README.md:25]:
```suggestion
**MeiliSearch Go** is the MeiliSearch API client for Go developers. **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, facets and synonyms are provided out-of-the-box.
```

Comment (curquiza) [reply README.md:25]:
```suggestion
**MeiliSearch Go** is the MeiliSearch API client for Go developers. **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, facets and synonyms are provided out-of-the-box.
```

Comment (curquiza) [reply README.md:25]: WDYT @eskombro and @bidoubiwa?

File: README.md (hunk 1 above)
Comment (curquiza) [reply README.md:23]: @bidoubiwa what do you think about my change here? Is it weird to you like the description below?

File: README.md (hunk 4 above)
Comment (eskombro) [reply README.md:25]: I like it! 🚀

**Comment by curquiza** [comment]
Merging!!!! 🚀

**Pull request merged**